NGINX_BASE

```diff
@@ -1 +1 @@
-registry.k8s.io/ingress-nginx/nginx:v20230623-427f3d2fb@sha256:7b479f66872c0b1cb0f1315e305b8a3e9c6da846c7dd3855db99bc8cfd6791e1
+rpkatz/nginx:withoutgeoip1
```

Comment:
Do we need to do a custom image build in CI？

Reply:
Actually we need 2 PRs:
* one removed geoip files and configs from ingress controller and nginx images
* this will kick a new image build. We need to promote it
* then we need to open another PR to change nginx base


In this PR (and that's why I've left the hold) we are simply using an Image I've built without geoip files first to test if the CI passes

Overall I want this intermediate images to be built on CI when changed, and consumed directly instead of promotions, etc. still under discussion with k8s github mgmt